src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_SwitchExpression.cs

```diff
@@ -20,14 +20,16 @@ public override BoundNode VisitConvertedSwitchExpression(BoundConvertedSwitchExp
             return SwitchExpressionLocalRewriter.Rewrite(this, node);
         }
 
-        private class SwitchExpressionLocalRewriter : BaseSwitchLocalRewriter
+        private sealed class SwitchExpressionLocalRewriter : BaseSwitchLocalRewriter
         {
             private SwitchExpressionLocalRewriter(BoundConvertedSwitchExpression node, LocalRewriter localRewriter)
-                : base(node.Syntax, localRewriter, node.SwitchArms.SelectAsArray(arm => arm.Syntax), isSwitchStatement: false)
+                : base(node.Syntax, localRewriter, node.SwitchArms.SelectAsArray(arm => arm.Syntax))
             {
             }
 
-            public static BoundExpression Rewrite(LocalRewriter localRewriter, BoundConvertedSwitchExpression node)
+            protected override bool IsSwitchStatement => false;
+
+            public static BoundExpression Rewrite(LocalRewriter localRewriter, BoundSwitchExpression node)
```

Comment:
>BoundSwitchExpression [](start = 79, length = 21)

Change to `BoundConvertedSwitchExpression` to get this to compile.

Reply:
Ah, bad merge.